packages/gensx/src/utils/bundler.ts

```diff
@@ -68,9 +86,54 @@ export async function bundleWorkflow(
     // This is handled by the build.sh script
     return path.join(outDir, "dist.tar.gz");
   } catch (error) {
-    console.error("Error bundling workflow", error);
+    console.error("Error bundling workflow with Docker", error);
     console.error("Stdout:\n", stdout);
     console.error("Stderr:\n", stderr);
     throw error;
   }
 }
+
+async function bundleLocally(
+  packageJsonDir: string,
+  relativeWorkflowPath: string,
+  outDir: string,
+): Promise<string> {
+  try {
+    // Create dist directory inside outDir
+    const distDir = path.join(outDir, "dist");
+    mkdirSync(distDir, { recursive: true });
+
+    // Move to the project directory
+    const originalDir = process.cwd();
+    process.chdir(packageJsonDir);
+
+    // Install @vercel/ncc if not already installed
+    try {
+      await execPromise("npx --no-install @vercel/ncc --version");
+    } catch (_error) {
+      await execPromise("npm install -g @vercel/ncc");
+    }
```

Comment:
Why do we need to force it to be installed globally? Does `npx @vercel/ncc` not work?

Reply:
if we didn't have global there, wouldn't we end up modifying the user's package.json with this dependency? We could do `npx --package @vercel/ncc ncc build ./${relativeWorkflowPath} -o ${distDir} --target es2022` to run it without installing it but that adds a few seconds each time. 

Perhaps another argument for doing away with the bundling.

Reply:
removed this code path altogether